scipy/stats/_distn_infrastructure.py

```diff
@@ -100,7 +100,7 @@
     (Differential) entropy of the RV.
 """
 _doc_fit = """\
-fit(data, %(shapes)s, loc=0, scale=1)
+fit(data, %(shapes)s)
     Parameter estimates for generic data.
```

Comment:
Something like "See `scipy.stats.rv_continuous.fit` (link) for detailed documentation of the keyword arguments." would be useful.

Reply:
Added